fargate/Dockerfile

```diff
@@ -0,0 +1,8 @@
+FROM dtandersen/factorio:latest
+
+RUN apk add --update python py-pip

```

Comment:
You should not update any packages if it isn't necessary and you forgot two flags.
I would use ``apk --no-cache --no-progress add py-pip``

Reply:
I'm not super familiar with APK (or APT, for that matter) but although there IS an APK awscli package, it's still unreleased (edge?) and I couldn't figure out how to install *that* instead. Otherwise, the python install of the AWS CLI is the simplest set of steps, even if it does balloon the image size :/

Reply:
That is apk from Alpine and not apt from Debian :).
I was more nitpicking about the ``--update``. I think it is fine to use the ``pip`´ variant but if you are interested you can find it here https://pkgs.alpinelinux.org/package/edge/testing/x86_64/aws-cli